<launch>
	
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find map_compression)/cfg/locTest.rviz" output="screen"/>


  	<node pkg="map_compression" type="locTest" name="locTest" output="screen">
	
	<!--KITTI or YQ # add Chery.vtk-->
	<param name="isChery" value="true" />	
	<param name="isKITTI" value="false" />	
	

	<param name="wholeMapName" value="/home/yh/mapModel/2019/07.23/loc/loc_park_01_learn.ply"/> 

	<param name="velodyneDirName" value="/media/yh/YH/DataSet/chery/scan/"/>
	<param name="icpFileName" value="/home/yh/mapModel/2019/07.23/park_pose.txt"/>

	<param name="startIndex" value="700"/>
	<param name="endIndex" value="1150"/>
	
	<param name="inputFilterYaml" value="$(find map_compression)/cfg/loc_laser_filters_park.yaml"/>
	<param name="mapFilterYaml" value="$(find map_compression)/cfg/loc_map_filters_park.yaml"/>
	<param name="icpYaml" value="$(find map_compression)/cfg/loc_icp_park.yaml"/>
	
	<param name="savePoseName" value="/home/yh/test.txt"/> 
	<param name="saveTimeName" value="/home/yh/test.txt"/> 
	<param name="saveIterName" value="/home/yh/test.txt"/> 


  </node>
</launch>
